poem/Poem/poem/templates/hints_metrics

```diff
@@ -0,0 +1,7 @@
+[
+  {% for metric in result %}
+		"{{ metric }}"
+		{% if not forloop.last %},
+		{% endif %}
+	{% endfor %} 
```

Comment:
Actually it's a django template language which is not indentation sensitive so it would have worked anyway